doc/lspconfig.txt

```diff
@@ -297,9 +297,6 @@ below returns a function that takes as its argument the current buffer path.
 - `util.find_git_ancestor`: a function that locates the first parent directory
   containing a `.git` directory. >
    root_dir = util.find_git_ancestor
-- `util.find_node_modules_ancestor`: a function that locates the first parent
```

Comment:
Let's leave some breadcrumbs, for now. We can decide later how to present/word it.

```
- Locate the first parent dir containing a "node_modules" dir: >
  vim.fs.find('node_modules', { path = root_dir, upward = true })
>
  - Replaces the old `util.find_node_modules_ancestor` API which is deprecated and will be removed.
```

Reply:
Tbh we already write that `find_...` is deprecated in favor of `vim.fs.find` so I guess I can just revert it and leave it be for now?

Reply:
The code snippet in the above example was helpful for me. It really helps not to have to track this stuff down later, why don't we just leave a breadcrumb? It is zero-cost.

Reply:
I think you misunderstood, I was agreeing with you about leaving a breadcrumb. Your suggestion sounds fine, let's go with that 👍